styles.css

```diff
@@ -69,13 +70,20 @@ h3 {
     margin: 0;
 }
 
-article, main {
+article, main, .back-button {
     margin: 40px auto;
     background-color: #FFFFFF;
     padding: 15px;
     border-radius: 5px;
 }
 
+.back-button {
+   width: 25%;
```

Comment:
it may be better to give this a px width, as it looks strange when it goes onto two lines. Can you use:
`width: -webkit-fit-content;
    width: -moz-fit-content;
    width: fit-content;`
that is in the ingredient classes?